services/ui-src/src/measures/2023/shared/CommonQuestions/OptionalMeasureStrat/ndrSets.tsx

```diff
@@ -128,11 +132,12 @@ const TotalNDRSets = ({
   return (
     <CUI.Box>
       <CUI.Divider key={`totalNDRDivider`} mt={2} mb={5} />
-      {categories.length > 0 && (
-        <CUI.Heading size={"sm"} key={`totalNDRHeader`}>
-          {totalQual}
-        </CUI.Heading>
-      )}
+      {categories.length > 0 &&
+        categories?.filter((item) => item.label).length > 0 && (
```

Comment:
Same suggestion as above, here

```suggestion
        categories.some((item) => item.label) && (
```